index.html

```diff
@@ -1,36 +1,106 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Awesome Books</title>
-	<link rel="stylesheet" href="index.css">
-</head>
-<body>
-  <main>
-	  
-		<section class="books">
-			<h1>Awesome Books</h1>
-		  <ul class="bookslist">
-			  
-		  </ul>
-		</section>
-
-		<div>
-		<div class="line"></div>
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <header>
+		<div class="nav-section">
+			<h3>Awesome</h3>
+			<nav class="nav-links">
+				<ul>
+					<li class="links list-section">List</li>
```

Comment:
- [ ] [OPTIONAL] It would be nice to have the content of the list displayed on the homepage instead of your homepage being empty